<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchExporterQtClass</class>
 <widget class="QMainWindow" name="BatchExporterQtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>858</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Nebula 3 Batch Exporter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>6</number>
    </property>
    <item row="1" column="0">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>2</number>
       </property>
       <item row="0" column="1" rowspan="3">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="3">
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QTextEdit" name="outputText">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="undoRedoEnabled">
             <bool>false</bool>
            </property>
            <property name="lineWrapMode">
             <enum>QTextEdit::WidgetWidth</enum>
            </property>
            <property name="lineWrapColumnOrWidth">
             <number>80</number>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="overwriteMode">
             <bool>true</bool>
            </property>
            <property name="cursorWidth">
             <number>0</number>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="exportButton">
            <property name="text">
             <string>Export</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" rowspan="3">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item row="6" column="0">
           <widget class="QCheckBox" name="exportAudio">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Audio</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="exportGameData">
            <property name="text">
             <string>Game Data</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="exportShaders">
            <property name="text">
             <string>Shaders</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="exportGraphics">
            <property name="text">
             <string>Assets</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Content</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="11" column="0">
           <widget class="QRadioButton" name="forceExport">
            <property name="text">
             <string>Force</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QSpinBox" name="jobs">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>16</number>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Parallel Jobs</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QProgressBar" name="progress">
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="status">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="toolkitLabel">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>6</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1027</width>
     <height>21</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>foo</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSetWorkDir"/>
    <addaction name="actionSetToolkitDirectory"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionContentbrowser"/>
   <addaction name="actionLeveleditor"/>
   <addaction name="actionSetToolkitDirectory"/>
   <addaction name="actionSetWorkDir"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionSetWorkDir">
   <property name="icon">
    <iconset resource="../../qttools/ui/qttoolkit.qrc">
     <normaloff>:/BlackIcons/icons/black/png/home_icon_48.png</normaloff>:/BlackIcons/icons/black/png/home_icon_48.png</iconset>
   </property>
   <property name="text">
    <string>Set working directory</string>
   </property>
  </action>
  <action name="actionSetNodyDirectory">
   <property name="text">
    <string>Set Nody directory</string>
   </property>
  </action>
  <action name="actionSetToolkitDirectory">
   <property name="icon">
    <iconset resource="../../qttools/ui/qttoolkit.qrc">
     <normaloff>:/BlackIcons/icons/black/png/wrench_plus_2_icon_48.png</normaloff>:/BlackIcons/icons/black/png/wrench_plus_2_icon_48.png</iconset>
   </property>
   <property name="text">
    <string>Set Toolkit directory</string>
   </property>
  </action>
  <action name="actionContentbrowser">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/icons/Contentbrowser.png</normaloff>:/icons/icons/Contentbrowser.png</iconset>
   </property>
   <property name="text">
    <string>Contentbrowser</string>
   </property>
   <property name="toolTip">
    <string>Start Contentbrowser</string>
   </property>
  </action>
  <action name="actionLeveleditor">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/icons/Leveleditor.png</normaloff>:/icons/icons/Leveleditor.png</iconset>
   </property>
   <property name="text">
    <string>Leveleditor</string>
   </property>
   <property name="toolTip">
    <string>Start Leveleditor</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../qttools/ui/qttoolkit.qrc"/>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
